Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -42,9 +47,17 @@ protected static IMixedRealityInputSystem InputSystem
             }
         }
 
-        // list of pointers
-        protected List<IMixedRealityPointer> pointers = new List<IMixedRealityPointer>();
-        public List<IMixedRealityPointer> Focusers => pointers;
+        protected List<IMixedRealityPointer> focusingPointers = new List<IMixedRealityPointer>();
+        /// <summary>
+        /// Pointers that are focusing the interactable
+        /// </summary>
+        public List<IMixedRealityPointer> Focusers => focusingPointers;
```

Comment:
By convention, properties are named as the field backing them but with initial capital letter. In this case it should be `FocusingPointers`. Same for `Pressers` below.

Reply:
Done.